view/theme/frio/templates/sub/direction.tpl

```diff
@@ -4,7 +4,7 @@
 	{{if $direction.direction == 1}}
 		<i class="fa fa-inbox" aria-hidden="true" title="{{$direction.title}}"></i>
 	{{elseif $direction.direction == 2}}
-		<i class="fa fa-download" aria-hidden="true" title="{{$direction.title}}"></i>
+		<i class="fa fa-comment-o" aria-hidden="true" title="{{$direction.title}}"></i>
```

Comment:
`2` is also used for indicating that the post had been fetched. So I suggest creating a direction "5" with the "comment" icon.

Reply:
Ok.